common/source/docs/common-gps-for-yaw.rst

```diff
@@ -96,7 +95,21 @@ The Holybro Unicore UM982 GPS should have the following parameters set:
 
 .. note:: this unit can be used with only its "Master" antenna connected, if desired, but no yaw information should be used. In this case set :ref:`GPS_TYPE<GPS_TYPE>` = 24 (UnicoreMaster)
 
-For the Holybro UM982 GPS, and other Unicore UM982 GPSes, the "Master" and "Slave" antennas have more mounting flexibility, but must be mounted at least 30cm apart on the vehicle. The offset distances in the x/y/z directions must be entered for the following parameters:
+For the Holybro UM982 GPS, and other Unicore UM982 GPSes like the CUAV C-RTK2 HP, the "Master" and "Slave" antennas have more mounting flexibility, but must be mounted at least 30cm apart on the vehicle. The offset distances in the x/y/z directions must be entered detailed in the Master-Slave Antenna Offsets section below.
```

Comment:
Amended to read: "For dual antenna systems such as C-RTK2 HP and Holybro UM982， "Master" and "Slave" antennas have more installation flexibility, but must be mounted at least 30cm apart on the vehicle. x/y/z The offset distance for the direction must be entered in detail in the master-slave antenna offset section below."

Reply:
thanks...will change